pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -486,19 +492,6 @@ func composePeerAuthentication(rootNamespace string, configs []*config.Config) *
 		}
 	}
 
-	if meshCfg == nil && namespaceCfg == nil && workloadCfg == nil {
-		// Return nil so that caller can fallback to apply alpha policy. Once we deprecate alpha API,
-		// this special case can be removed.
-		return nil
```

Comment:
Could you also update other places (e.g. `getMutualTLSModeForPort`) that are using the result from here so that they do not need to check nil now?

Reply:
Good catch, removed related nil checks, thanks!